src/SprykerEco/Yves/AmazonPay/Dependency/Client/AmazonPayToShipmentBridge.php

```diff
@@ -31,6 +31,16 @@ public function __construct($shipmentClient)
      */
     public function getAvailableMethods(QuoteTransfer $quoteTransfer)
     {
+        if (method_exists($this->shipmentClient, 'getAvailableMethodsByShipment') === true) {
+
+            $shipmentMethodsCollectionTransfer = $this->shipmentClient->getAvailableMethodsByShipment($quoteTransfer);
+
+            $shipmentMethodsTransfer = $shipmentMethodsCollectionTransfer->getShipmentMethods()->getIterator()
+                ->current();
```

Comment:
Please add a `count() > 1` check and throw an exception that multiple shipments are not supported in this version of the module.

Reply:
added